src/scales/scale.time.js

```diff
@@ -755,7 +755,7 @@ module.exports = Scale.extend({
 		var me = this;
 		var ticksOpts = me.options.ticks;
 		var tickLabelWidth = me.ctx.measureText(label).width;
-		var angle = helpers.toRadians(ticksOpts.maxRotation);
+		var angle = helpers.toRadians(me.isHorizontal() ? ticksOpts.maxRotation : ticksOpts.minRotation);
```

Comment:
> `maxRotation` defaults to 50, but vertical axis ticks are only rotated to `minRotation`

This is a correct assumption? [Document](https://www.chartjs.org/docs/master/axes/cartesian/#tick-configuration) says `maxRotation` and `minRotation` are only applicable to horizontal scales.

Reply:
Example: https://codepen.io/kurkle/pen/rggrEG

Reply:
I think it is not completely supported yet. Below is the result with `position: 'right'`. Need to adjust `paddingTop` and `paddingBottom` at the end of `scale.fit`?
<img width="159" alt="Screen Shot 2019-06-06 at 5 25 28 PM" src="https://user-images.githubusercontent.com/723188/59022322-20dc2300-8880-11e9-92cc-60cc47c3619c.png">

Reply:
Also, currently `scale.labelRotation` is set only for horizontal scales in `calculateTickRotation`.

Reply:
Agreed, but I'd leave those to another PR.